analyzer/javatests/com/engflow/bazel/invocation/analyzer/suggestionproviders/BottleneckSuggestionProviderTest.java

```diff
@@ -288,4 +292,50 @@ public void createsOnlyLimitedSuggestions()
     assertThat(suggestions.hasFailure()).isFalse();
     assertThat(suggestions.getMissingInputList()).isEmpty();
   }
+
+  @Test
+  public void doNotAddCaveatIfTargetNamesAreNotIncludedAndThereAreNoBottlenecks()
+      throws InvalidProfileException, MissingInputException, NullDatumException {
+    when(dataManager.getDatum(FlagValueExperimentalProfileIncludeTargetLabel.class))
+        .thenReturn(new FlagValueExperimentalProfileIncludeTargetLabel(true));
+    when(dataManager.getDatum(TotalDuration.class))
+        .thenReturn(new TotalDuration(Duration.ofSeconds(100)));
+    when(dataManager.getDatum(EstimatedCoresUsed.class)).thenReturn(new EstimatedCoresUsed(4, 0));
+    when(dataManager.getDatum(ActionStats.class)).thenReturn(new ActionStats(List.of()));
+
+    final var bottleneckSuggestionProvider =
+        new BottleneckSuggestionProvider(1, 1, Duration.ZERO, 0, 1.);
```

Comment:
here and below:

```suggestion
        new BottleneckSuggestionProvider(1, 1, Duration.ZERO, 0, 1);
```

the trailing `.` is unnecessary; the function signature declares this param to be `double`, so it will be a double no matter waht

Reply:
This comes from c&p other test cases. I can adjust all of them to stay consistent, though.

Reply:
As you wish.